src/app/client/src/app/modules/dashboard/components/filter/filter.component.spec.ts

```diff
@@ -202,4 +202,20 @@ describe('FilterComponent', () => {
     expect(component.filtersFormGroup.get('Grade').value).toEqual(['08-01-2019', '09-01-2019', '10-01-2019']);
   }));
 
+  it('should update form', fakeAsync(() => {
+    component.ngOnInit();
+    tick(1000);
```

Comment:
Please revisit this and write it in accordance with the use case of the function. Whenever you set the value in the form control form update should be called and do its job. Also use more specific assertions

Reply:
resolved